GERTi/GERTi Client.lua

```diff
@@ -162,22 +180,26 @@ handler."OPENROUTE" = local function RouteHandler(eventName, receivingModem, sen
             end
         end
     end
+
+    return true
```

Comment:
OpenRoute is always going to be returning something else, so the return true is unnecessary.